src/SignalR/clients/cpp/src/signalrclient/Build/VS/signalrclient.vcxproj

```diff
@@ -17,16 +17,16 @@
   <Import Project="..\..\..\..\Build\Config.Definitions.props" />
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v140</PlatformToolset>
```

Comment:
Why downgrade from 141 to 140?

Reply:
Would be preferable to keep the toolset version consistent across all cpp projects in the repo. Maybe we should enforce this via eng/targets/Cpp.Common{props.targets} ?

Reply:
Hopefully a temporary change. If you look at the C++ packages we restore, they are meant to run with v140. I also thought it was causing the build issues, turns out it was other things.